remote.go

```diff
@@ -1148,6 +1148,46 @@ func (wd *remoteWD) execScript(script string, args []interface{}, suffix string)
 	return reply.Value, nil
 }
 
+func (wd *remoteWD) execCdpCommandRaw(data []byte) ([]byte, error) {
+	return wd.execute("POST", wd.requestURL("/session/%s/goog/cdp/execute", wd.id), data)
+}
+
+// execCdpCommand execute cdp command
+// this interface didn't define in official wiki(https://github.com/SeleniumHQ/selenium/wiki/JsonWireProtocol)
+// so I just refered to selenium in python (selenium.webdriver.chrome.remote_connection.ChromeRemoteConnection)
+func (wd *remoteWD) execCdpCommand(cmd string, params map[string]interface{}) (interface{}, error) {
```

Comment:
In Go, initialisms are written in the same case. Here, change this function name to "execCDPCommand".

https://github.com/golang/go/wiki/CodeReviewComments#initialisms

But actually, since the below function is part of the exported API, we should be more explicit. Perhaps we should call this "execChromeDPCommand" (and similar below) to make the fact that this is a Chrome-specific method.